XBOFS.win.sln

```diff
@@ -628,6 +629,7 @@ Global
 		{22E13D1D-2BBC-4588-98FF-8A4B62DB5922} = {ADD10A5F-41E1-4F1C-869D-FBBDECA4574A}
 	EndGlobalSection
 	GlobalSection(ExtensibilityGlobals) = postSolution
+		Qt5Version = $(DefaultQtVersion)
```

Comment:
@plbpietrz Hmmmm, `Qt5Version`?

Reply:
VC++ qt integration magic probably, will check if this can be cleaned. As for testing, have the newest vcpkg, cleaned of libraries if possible, _integrated_ with VC and just build the solution. During compilation, vcpkg should now install the required versions of dependencies to a local dir (mentioned in .gitignore).

Reply:
Still builds without `Qt5Version = $(DefaultQtVersion)`

Reply:
@plbpietrz Nice. I'll try and give this a quick test run tomorrow morning. Assuming all works well, I'll merge it :-)

Reply:
@plbpietrz Going to be a bit of extra delay on the merge, been a bit busy on my end, still need to test it :-/

Reply:
@OOPMan sure, no problem. I've also enabled precompiled headers on the XBOFS.win subproject for faster recompilation, no changes to the code just a tweak to includes and config, so if your interested I can also push that.

Reply:
@plbpietrz Feel free to :-)

Reply:
@OOPMan an additional question, I noticed that when logging in some cases u use wide `L""` strings instead of normal `char`, is that intentional? If not I was thinking of making them uniformly encoded using `tchar.h` `_T` macro which in the current way the project is setup would unwind to `L""` strings.

Reply:
@plbpietrz IIRC, I did that due to spdlog not being super happy with the tchar stuff when I tried it. You're welcome to try and get it working, but if you start getting weird crashes it might be due to that. 

I ran into an odd issue on my end trying to compile. See the following screenshot:

![image](https://user-images.githubusercontent.com/248980/155007355-981d9a2f-0346-4e7b-842b-bdb145db3891.png)

The first relates to the line of code you mentioned in the issue you posted:
` if (resultHandle != S_OK || deviceInterfaceList == TEXT('\0')) {`

The other relates to this line in `main.cpp`:
` SingleInstanceLimiter limiter(TEXT("Global\\{74cbcbe6-4202-4a85-9362-ae371291abd6}"));`

That's interesting is that when I tested on my unmodified master branch of the application the same issue happens. The only thing I can think is that whatever VS2019 update I installed before checking your branch changed something. Have you seen this, or are you able to replicate it?

Reply:
I'm currently on the latest VS2019 patch and only the `deviceInterfaceList` copped up in my builds. I pushed the precompiled headers change also with the pointer fix.

Reply:
@plbpietrz Thanks. I just realised I hadn't switched to the branch for the MR 🤣 Tested on the right branch and everything seems good. Don't know if you want to add anything additional?

Reply:
@OOPMan ok, I've noticed that I was building with Win32 configuration, that caused XBOFS.win.qt to be skipped. Using x64 config the project started building but hit similar problem as you with the `SingleInstanceLimiter`

Reply:
@OOPMan Well I pushed one missing include just 10min ago that solved a issue on my end so if everything is building on your end on the latest commit then I'm all good 😄

Reply:
I can also add the update to `SingleInstanceLimiter` to this MR so that it all compiles nicely on the newest VS2019

Reply:
@plbpietrz Sure, feel free, although it seems to have compiled fine when I tested just now :-)